base_location_geonames_import/__manifest__.py

```diff
@@ -9,22 +8,28 @@
 
 {
     'name': 'Base Location Geonames Import',
-    'version': '11.0.1.1.1',
+    'version': '12.0.1.1.1',
     'category': 'Partner Management',
     'license': 'AGPL-3',
-    'summary': 'Import better zip entries from Geonames',
+    'summary': 'Import zip entries from Geonames',
     'author': 'Akretion,'
               'Agile Business Group,'
-              'Antiun Ingeniería S.L.,'
+              u'Antiun Ingeniería S.L.,'
               'Tecnativa,'
               'Odoo Community Association (OCA)',
-    'website': 'http://www.akretion.com',
-    'depends': ['base_location'],
-    'external_dependencies': {'python': ['requests']},
+    'website': 'https://github.com/OCA/partner-contact',
+    'depends': [
+        'base_location'
+    ],
+    'external_dependencies': {
+        'python': [
+            'requests'
```

Comment:
This one is a requisite of Odoo itself, so no need to add it again:

https://github.com/odoo/odoo/blob/50b506996dc0308dc6b17ec6b96d07d9ad81fc69/requirements.txt#L36